### PR TITLE
Rev 52 UI tweaks

### PR DESCRIPTION
**Pull request by IbrahimAljabrST**
### Jira URL

https://optimumpartners.atlassian.net/browse/REV-52

### Description

Add hover for cards, fix the home page

### Dependencies in Other PRs,Repo

none

### Testing Instructions

open home page test the hover and styling

### CODE REVIEW

File: src/components/home/FeaturedListingCard/featured-listing-card.styles.js [hunk 1]
```diff
@@ -37,35 +94,66 @@ export const IconsContainer = styled(Container)`
 
 export const CardImageContainer = styled.div`
   position: relative;
+  overflow: hidden;
+  height: 12rem;
+  ${CardContainer}:hover & {
```
Comment (AhmadAlShwaiki) [inline src/components/home/FeaturedListingCard/featured-listing-card.styles.js:99]:
```suggestion

  ${CardContainer}:hover & {
```

File: src/components/home/FeaturedListingCard/featured-listing-card.styles.js [hunk 2]
```diff
@@ -37,35 +94,66 @@ export const IconsContainer = styled(Container)`
...
+    height: 100%;
+  }
 `;
 
 export const Image = styled.img`
-  height: 18rem;
+  height: 12rem;
   width: 100%;
+  transition: 0.2s;
+  ${CardContainer}:hover & {
```
Comment (AhmadAlShwaiki) [inline src/components/home/FeaturedListingCard/featured-listing-card.styles.js:108]:
```suggestion

  ${CardContainer}:hover & {
```

File: src/components/home/FeaturedListingCard/featured-listing-card.styles.js [hunk 3]
```diff
@@ -84,27 +177,32 @@ export const PersonImg = styled.img`
...
 export const TextMedium = styled.p`
   color: ${colors.mineShaft};
   font-size: 1rem;
-  margin: 0.1875rem 0 0.9375rem 0;
+  margin: 0.2rem 0 0 0;
   line-height: 1.2rem;
   font-weight: 600;
 `;
 
 export const TextSmall = styled.p`
```
Comment (AhmadAlShwaiki) [inline src/components/home/FeaturedListingCard/featured-listing-card.styles.js:202]: Please remove this component and the above as well, create one with their shared css props and pass in whatever different values are for the rest.

File: src/components/shared/Footer/footer.styles.js [hunk 4]
```diff
@@ -77,9 +77,25 @@ export const SubscribeWrapper = styled.div`
   color: ${colors.green};
 `;
 
-export const SubscribeInputWrapper = styled.div`
-  height: 2.375rem;
+export const LogoContainer = styled.div`
+  svg {
```
Comment (AhmadAlShwaiki) [inline src/components/shared/Footer/footer.styles.js:81]: This can be in the parent, delete this component please

**Review by AhmadAlShwaiki — CHANGES_REQUESTED** [review]
Some css adjustments, looks really good overall

File: src/components/home/FeaturedListingCard/featured-listing-card.styles.js [hunk 5]
```diff
@@ -140,3 +179,53 @@ export const IconContainer = styled.div`
...
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  border-radius: 0.3rem;
+`;
+
+export const CardParents = styled.div`
```
Comment (AhmadAlShwaiki) [inline src/components/home/FeaturedListingCard/featured-listing-card.styles.js:216]:
```suggestion
export const CardParent = styled.div`
```

File: src/components/home/FeaturedListingCard/featured-listing-card.styles.js [hunk 6]
```diff
@@ -140,3 +179,53 @@ export const IconContainer = styled.div`
...
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  display: none;
+  position: absolute;
+  border-radius: 0.3rem;
+  overflow: hidden;
+`;
+
+export const Overlay = styled.div`
+  background-color: ${colors.mineShaft}99;
```
Comment (AhmadAlShwaiki) [inline src/components/home/FeaturedListingCard/featured-listing-card.styles.js:206]:
```suggestion
```

Comment (AhmadAlShwaiki) [reply src/components/home/FeaturedListingCard/featured-listing-card.styles.js:206]: Please remove any repeated css properties below as well.

Comment (IbrahimAljabrST) [reply src/components/home/FeaturedListingCard/featured-listing-card.styles.js:206]: this is the overlay

**Review by AhmadAlShwaiki — APPROVED** [review]

**Pull request merged**